corehq/apps/app_manager/views/utils.py

```diff
@@ -352,20 +352,21 @@ def update_linked_app(app, master_app_id_or_build, user_id):
     ):
         old_multimedia_ids = set([media_info.multimedia_id for path, media_info in app.multimedia_map.items()])
         report_map = get_static_report_mapping(master_build.domain, app['domain'])
-        if not app.domain_link.is_remote:
-            report_map.update({
-                c.report_meta.master_id: c._id
-                for c in get_report_configs_for_domain(app.domain)
-                if c.report_meta.master_id
-            })
+
+        report_map.update({
+            c.report_meta.master_id: c._id
+            for c in get_report_configs_for_domain(app.domain)
+            if c.report_meta.master_id
+        })
```

Comment:
I am not well aware of how linked UCRs work and my understanding was that this won't work for remote links because master_id won't be set.
If you say this would, i believe you.

Reply:
Yes, it gets set here:
https://github.com/dimagi/commcare-hq/blob/fr%2Fremote-linked-ucr/corehq/apps/linked_domain/ucr.py#L91

Reply:
Thanks. All of this looks okay but I would still defer to someone who understands linked UCRs better to drop an approval.

Great to see that this solves issues with updating remotely linked apps as well!